projects/1.20/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -0,0 +1,874 @@
+  "item.pamhc2foodextended.misopasteitem": "味噌",
+  "item.pamhc2foodextended.misosoupitem": "味噌汤",
+  "item.pamhc2foodextended.mixedflowerssaladitem": "繁花沙拉",
+  "item.pamhc2foodextended.mixedsaladitem": "什锦沙拉",
+  "item.pamhc2foodextended.mochaicecreamitem": "摩卡冰淇淋",
+  "item.pamhc2foodextended.mochicakeitem": "麻薯蛋糕",
+  "item.pamhc2foodextended.mochidessertitem": "麻薯",
+  "item.pamhc2foodextended.mochiitem": "麻薯",
+  "item.pamhc2foodextended.molasseschickenitem": "蜜汁鸡肉",
+  "item.pamhc2foodextended.mozzerellasticksitem": "奶酪棒",
+  "item.pamhc2foodextended.mulberryjellyitem": "桑葚酱",
+  "item.pamhc2foodextended.mulberryjellysandwichitem": "桑葚酱三明治",
+  "item.pamhc2foodextended.mulberryjellytoastitem": "桑葚酱吐司",
+  "item.pamhc2foodextended.mulberryjuiceitem": "桑葚汁",
+  "item.pamhc2foodextended.mulberrypieitem": "桑葚派",
+  "item.pamhc2foodextended.mulberrysmoothieitem": "桑葚沙冰",
+  "item.pamhc2foodextended.mulberryyogurtitem": "桑葚酸奶",
+  "item.pamhc2foodextended.museliitem": "杂锦果麦",
+  "item.pamhc2foodextended.mushroomketchupitem": "蘑菇沙司",
+  "item.pamhc2foodextended.mushroomketchupomeletitem": "蘑菇酱煎蛋饼",
+  "item.pamhc2foodextended.mushroomlasagnaitem": "蘑菇千层面",
+  "item.pamhc2foodextended.mushroomoilitem": "蘑菇油",
+  "item.pamhc2foodextended.mushroomrisottoitem": "牛肝菌烩饭",
+  "item.pamhc2foodextended.mushroomsteakitem": "蘑菇牛排",
+  "item.pamhc2foodextended.mustarditem": "芥末",
+  "item.pamhc2foodextended.musubiitem": "夏威夷火腿饭团",
+  "item.pamhc2foodextended.naanitem": "印度烤饼",
+  "item.pamhc2foodextended.nachoesitem": "干酪玉米片",
+  "item.pamhc2foodextended.neapolitanicecreamitem": "那不勒斯冰淇淋",
+  "item.pamhc2foodextended.nopalessaladitem": "诺帕莱斯沙拉",
+  "item.pamhc2foodextended.nuttoppedicecreamitem": "“尖”果冰淇淋",
+  "item.pamhc2foodextended.oatmealraisincookiesitem": "燕麦葡萄干饼干",
+  "item.pamhc2foodextended.okrachipsitem": "秋葵片",
+  "item.pamhc2foodextended.okracreoleitem": "奥尔秋葵",
+  "item.pamhc2foodextended.oldworldveggiesoupitem": "古早味蔬菜汤",
+  "item.pamhc2foodextended.omeletitem": "煎蛋饼",
+  "item.pamhc2foodextended.omuriceitem": "蛋包饭",
+  "item.pamhc2foodextended.onigiriitem": "饭团",
+  "item.pamhc2foodextended.onionhamburgeritem": "洋葱汉堡",
+  "item.pamhc2foodextended.onionringsitem": "洋葱圈",
+  "item.pamhc2foodextended.orangechickenitem": "陈皮鸡",
+  "item.pamhc2foodextended.orangegingerbeefitem": "橙香生姜牛肉",
+  "item.pamhc2foodextended.orangejellyitem": "橙子酱",
+  "item.pamhc2foodextended.orangejellysandwichitem": "橙子酱三明治",
+  "item.pamhc2foodextended.orangejellytoastitem": "橙子酱吐司",
+  "item.pamhc2foodextended.orangejuiceitem": "橙子汁",
+  "item.pamhc2foodextended.orangepieitem": "橙子派",
+  "item.pamhc2foodextended.orangesmoothieitem": "橙子沙冰",
+  "item.pamhc2foodextended.orangesodaitem": "橙子汽水",
+  "item.pamhc2foodextended.orangeyogurtitem": "橙子酸奶",
+  "item.pamhc2foodextended.ovenroastedcaulifloweritem": "烤花菜",
+  "item.pamhc2foodextended.oystersauceitem": "蚝油",
+  "item.pamhc2foodextended.padthaiitem": "泰式炒河粉",
+  "item.pamhc2foodextended.pambitsboxitem": "“潘”位音盒",
```

```suggestion
  "item.pamhc2foodextended.pambitsboxitem": "Pam点心饼干盒",
```
https://www.gritsbits.com/product-category/grits-bits/grits-bits-boxes/